plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -120,6 +121,27 @@ def package_headers(filename):
     return headers
 
 
+def package_headers_drpm(filename):
```

Comment:
I am not sure if it is correct to say drpm header, as far as i understood  dprm package has rpm header and just some info about old nevr, new nevr and sequence, so maybe just call this method drpm_package_info() or something similar?